temporal-sdk/src/main/java/io/temporal/nexus/NexusOperationContext.java

```diff
@@ -39,4 +40,10 @@ public interface NexusOperationContext {
    * WorkflowServiceStubsOptions.Builder#setMetricsScope(Scope)} when a worker starts up.
    */
   Scope getMetricsScope();
+
+  /**
+   * Get a {@link WorkflowClient} that can be used to start interact with the Temporal service from
+   * a Nexus handler.
+   */
+  WorkflowClient getWorkflowClient();
```

Comment:
Maybe add a method to get the nexus context too?

Reply:
If a use case comes up yes, at least internally.